src/Student5/Ex2/Body.js

```diff
@@ -0,0 +1,17 @@
+import React from "react";
+import "./Body.scss";
+
+export const Body = () => {
+  return (
+    <div className="body">
```

Comment:
Instead of creating one component called Body, I think it would be better to create two components: Posts, and Groups.  We want our React components to be small and focused.